vignettes/workflows.Rmd

```diff
@@ -0,0 +1,292 @@
+---
+title: "Basic workflows for talkr"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{workflow}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.width = 8
+)
+```
+
+```{r setup}
+library(talkr)
+```
+
+## Introduction
+`talkr` is a package designed for working with conversational data in R. 
+
+## Loading some data
+
+We will be using the IFADV corpus as example data for the workflow of `talkr`. This is a corpus consisting of 20 dyadic conversations in Dutch, published by the Nederlandse Taalunie in 2007 ([source](https://fon.hum.uva.nl/IFA-SpokenLanguageCorpora/IFADVcorpus/))
+
+The snippet below initializes the talkr dataset using the ifadv data. For more information about the IFADV dataset, see the [repository link](https://github.com/elpaco-escience/ifadv).
+
+
+```{r}
+
+data <- init(ifadv::ifadv) 
+
+```
+
+Essential to any `talkr` workflow is a minimal set of data fields. These are the following: \* `source`: the source conversation (a corpus can consist of multiple sources) \* `begin`: begin time (in ms) of an utterance \* `end`: end time (in ms) of an utterance \* `utterance`: content of an utterance \* `participant`: the person who produced the utterance
+
```

Comment:
Maybe this can be a list?
```suggestion
Essential to any `talkr` workflow is a minimal set of data fields. These are the following:

- `source`: the source conversation (a corpus can consist of multiple sources)
- `begin`: begin time (in ms) of an utterance
- `end`: end time (in ms) of an utterance
- `utterance`: content of an utterance
- `participant`: the person who produced the utterance

```